resolwe/flow/executors/prepare.py

```diff
@@ -40,18 +40,14 @@ def extend_settings(self, data_id, files):
 
     def get_tools(self):
```

Comment:
Can we take this opportunity to rename this to `get_tools_paths`?

We discussed this in the famous #308.